matchbox_socket/src/webrtc_socket/wasm/message_loop.rs

```diff
@@ -297,18 +321,19 @@ async fn try_add_rtc_ice_candidate(connection: &RtcPeerConnection, candidate_str
 async fn handshake_accept(
     signal_peer: SignalPeer,
     mut signal_receiver: UnboundedReceiver<PeerSignal>,
-    messages_from_peers_tx: UnboundedSender<(PeerId, Packet)>,
+    messages_from_peers_tx: Vec<UnboundedSender<(PeerId, Packet)>>,
     config: &WebRtcSocketConfig,
-) -> Result<(PeerId, RtcDataChannel), Box<dyn std::error::Error>> {
+) -> Result<(PeerId, Vec<RtcDataChannel>), Box<dyn std::error::Error>> {
     debug!("handshake_accept");
 
     let conn = create_rtc_peer_connection(config);
     let (channel_ready_tx, mut channel_ready_rx) = futures_channel::mpsc::channel(1);
```

Comment:
Now that this channel sends multiple messages, aren't we creating some chance that it will overflow? Should we increase capacity to number of channels?

...or make it only send a message when all channels are ready?

Reply:
fixed that - every channel should only send a single message now. I even tried to change it to a `oneshot` channel but I did not understand how they work - they don't seem to implement `next()`